### DISCUSSION TITLE
Blocked users will not be searched

### DISCUSSION THREAD

**Discussion by RSS1102** [community/community#127017] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When I block a user, I want them not to be indexed when I search for code snippets or repo names。

**Comment by LiteBrite82** [comment]
Hi @RSS1102 ,

Interesting. Can you provide some more detail? 
- What kind of account are you blocking them from?
- Are you trying to remove references to code they collaborated on?
- Can you provide a screenshot of the search results; show the information you want to have removed?
